CHANGELOG.md

```diff
@@ -1,5 +1,62 @@
 # Change Log
 
+## [0.8.0](https://github.com/angelozerr/lsp4xml/milestone/9?closed=1) (July 23, 2019)
+
+### Enhancements
+
+ * Find definition for start/end tag element. See[#535](https://github.com/angelozerr/lsp4xml/issues/535)
+ * Improve Source information from completion. See[#529](https://github.com/angelozerr/lsp4xml/issues/529)
+ * `XSD CodeLens` references should be clickable. See[#490](https://github.com/angelozerr/lsp4xml/issues/490)
+ * Add support for `textDocument/documentHighlight` for XML Schema types. See[#470](https://github.com/angelozerr/lsp4xml/issues/470)
+ * Add support for `textDocument/completion` for xs:element/@name / xs:extension/@base. See[#451](https://github.com/angelozerr/lsp4xml/issues/451)
+ * Support selective enablement of outline per file. See[#427](https://github.com/angelozerr/lsp4xml/issues/427)
```

Comment:
I would suggest: "Add support for selective outline enablement per file"

Reply:
@xorye updated